exercises/card.js

```diff
@@ -14,6 +14,7 @@ class Card{
 
 class Deck{
 	constructor(){
+		
```

Comment:
If you're going to use variables in your Deck object, they should be predefined here to make it easier to see